WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListFragment.java

```diff
@@ -213,14 +214,26 @@ public void onClick(View v) {
                                         }
                                     }).show();
                         } else if (savedLocally) {
-                            Snackbar.make(getActivity().findViewById(R.id.coordinator),
-                                    R.string.editor_post_saved_locally_not_published, Snackbar.LENGTH_LONG)
-                                    .setAction(R.string.button_publish, new View.OnClickListener() {
-                                        @Override
-                                        public void onClick(View v) {
-                                            publishPost(post);
-                                        }
-                                    }).show();
+                            if (isScheduledPost) {
```

Comment:
Yes, I did :)

Refactored in 406844fb1b30625ecd02a5fe607055329f7c5106. I considered shortening it a little further, but I think that crosses the "easy to understand" line, so I think this is good enough. What do you think?